[2025-03-19T12:48:13.902+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fraud_detection_pipeline.store_data_hdfs scheduled__2023-10-12T07:00:00+00:00 [queued]>
[2025-03-19T12:48:13.918+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fraud_detection_pipeline.store_data_hdfs scheduled__2023-10-12T07:00:00+00:00 [queued]>
[2025-03-19T12:48:13.919+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 2
[2025-03-19T12:48:13.943+0000] {taskinstance.py:1380} INFO - Executing <Task(BashOperator): store_data_hdfs> on 2023-10-12 07:00:00+00:00
[2025-03-19T12:48:13.951+0000] {standard_task_runner.py:57} INFO - Started process 11018 to run task
[2025-03-19T12:48:13.956+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'fraud_detection_pipeline', 'store_data_hdfs', 'scheduled__2023-10-12T07:00:00+00:00', '--job-id', '915', '--raw', '--subdir', 'DAGS_FOLDER/dag_1.py', '--cfg-path', '/tmp/tmptbyqg0jh']
[2025-03-19T12:48:13.961+0000] {standard_task_runner.py:85} INFO - Job 915: Subtask store_data_hdfs
[2025-03-19T12:48:14.033+0000] {task_command.py:415} INFO - Running <TaskInstance: fraud_detection_pipeline.store_data_hdfs scheduled__2023-10-12T07:00:00+00:00 [running]> on host 0009f5fb76aa
[2025-03-19T12:48:14.156+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='fraud_detection_pipeline' AIRFLOW_CTX_TASK_ID='store_data_hdfs' AIRFLOW_CTX_EXECUTION_DATE='2023-10-12T07:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-10-12T07:00:00+00:00'
[2025-03-19T12:48:14.159+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-03-19T12:48:14.161+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'docker cp /tmp/transactions.json namenode:/tmp/transactions.json && docker cp /tmp/customers.json namenode:/tmp/customers.json && docker cp /tmp/external_data.json namenode:/tmp/external_data.json && rm -rf /tmp/external_data.json  && rm -rf /tmp/customers.json  && rm -rf /tmp/transactions.json  && docker exec namenode bash -c "hdfs dfs -mkdir -p /data && hdfs dfs -put -f /tmp/transactions.json /data/transactions.json && hdfs dfs -put -f /tmp/customers.json /data/customers.json && hdfs dfs -put -f /tmp/external_data.json /data/external_data.json && rm -rf /tmp/customers.json  && rm -rf /tmp/customers.json  && rm -rf /tmp/external_data.json "']
[2025-03-19T12:48:14.176+0000] {subprocess.py:86} INFO - Output:
[2025-03-19T12:48:18.260+0000] {subprocess.py:93} INFO - 2025-03-19 12:48:18,257 INFO sasl.SaslDataTransferClient: SASL encryption trust check: localHostTrusted = false, remoteHostTrusted = false
[2025-03-19T12:48:20.519+0000] {subprocess.py:93} INFO - 2025-03-19 12:48:20,517 INFO sasl.SaslDataTransferClient: SASL encryption trust check: localHostTrusted = false, remoteHostTrusted = false
[2025-03-19T12:48:23.082+0000] {subprocess.py:93} INFO - 2025-03-19 12:48:23,079 INFO sasl.SaslDataTransferClient: SASL encryption trust check: localHostTrusted = false, remoteHostTrusted = false
[2025-03-19T12:48:23.739+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-03-19T12:48:23.780+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=fraud_detection_pipeline, task_id=store_data_hdfs, execution_date=20231012T070000, start_date=20250319T124813, end_date=20250319T124823
[2025-03-19T12:48:23.923+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-03-19T12:48:23.951+0000] {taskinstance.py:2776} INFO - 1 downstream tasks scheduled from follow-on schedule check
